### DISCUSSION TITLE
Custom local email templates are mounted as folder in kong

### DISCUSSION THREAD

**Discussion by peerknife** [supabase/supabase#30214] [Questions] [answered]
Hello,

I'm not able to get my custom email templates working on my local environment.
The e-mail that I receive in inbucket is an html file that shows the root of the the mounted folder.
<img width="668" alt="Screenshot 2024-10-31 at 21 03 57" src="https://github.com/user-attachments/assets/8f705d3b-4b72-4ca7-a7e0-5518b11f6493">

This is the entry in my `config.toml` and template itself:
```
...
[auth.email.template.confirmation]
subject = "Confirm Your Email"
content_path = "./supabase/auth/email/confirmation.html"
...
```

```
<h2>Sign up confirmation</h2>

<p>Thank you for signing up! To complete the registration process and gain access to our platform, please click the link below to confirm your account:</p>

<p><a href="{{ .SiteURL }}/auth/confirm?token_hash={{ .TokenHash }}&type=email">Confirm your email address</a></p>
```

When I look at the kong image, the html files are mounted, however when using the terminal, the `confirmation.html` is a folder, not a file.
<img width="1013" alt="Screenshot 2024-10-31 at 21 12 52" src="https://github.com/user-attachments/assets/0cc4e837-eec2-4669-b20d-b8e528602b3e">

Project & versions:
Sveltekit project

```
package.json
"@supabase/ssr": "^0.5.1",
"@supabase/supabase-js": "^2.46.1"
```

supabase version: 1.207.9

**Comment by peerknife** [comment]
Alright, after doing some more research (took me 1 day already), I figured it out (to get it at least working). The relative path wasn't working, when I changed it to an absolute path the email I received contained the correct template. I'm on mac and installed Supabase via homebrew.

Why is isn't it mounting the file correctly from where the Supabase service is started from (in my project root folder)? Maybe I missed something but this behaviour feels incorrect, or there is some important information missing in the docs.

**Reply by silentworks** [reply]
It should be using the relative path. What version of the Supabase CLI are you using?

**Reply by peerknife** [reply]
I’m using the latest version: 1.207.9. I switched installing the cli from homebrew to npm and got it working for a few tries, but after stopping and starting supabase after a few times I’m back in the old situation again, strangely enough, not sure what causes it.

I tried to remove the node_modules folder and install dependencies again but without success.

So in the end I’m still in the situation that relative path isn’t working but absolute is.

**Reply by peerknife** [reply]
I think that when it worked for a few times (after freshly installing supabase via npm), when starting supabase it mentioned the working directory location in the terminal, now I don’t see that anymore.

**Comment by peerknife** [comment, marked as answer]
This issue can be closed as I found what the problem was, I use Warp terminal which stupidly enough accepts the lower case version of 
 `/Users` . So instead of being in `/Users/...`I was in `/users/...` which warp could resolve but the docker not.

Closing this as this was the issue and now the relative paths are resolving perfectly. Thanks @silentworks for taking the time to read my question which luckily solved itself.
